bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -222,11 +241,9 @@ fn load_configs() -> (AwsConfig, Arc<Config>) {
     (aws_config, config)
 }
 
-fn enable_logging_subsystem(config: &Arc<Config>) {
-    let env_filter = format!(
-        "h2=off,hyper=off,rustls=off,datadog-trace-mini-agent=off,{:?}",
-        config.log_level
-    );
+fn enable_logging_subsystem() {
+    let env_filter =
+        "h2=off,hyper=off,reqwest=off,rustls=off,datadog-trace-mini-agent=off".to_string();
```

Comment:
might be worth to add `h1` too

Reply:
h1 is not a library

Reply:
Got carried over by thinking h2 = http2, h1 = http1

You're right